docs/plugins/transform-react-jsx-source.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+title: React JSX source transform
+description:
+permalink: /docs/plugins/transform-react-jsx-source/
+package: babel-plugin-transform-react-jsx-source
+---
+
+Adds source file and line number to JSX elements.
+
+## Example
+
+**In**
+
+```javascript
+// this/file.js
+<sometag />
+```
+
+**Out**
+
+```javascript
+var _jsxFileName = "this/file.js";
```

Comment:
IMHO it's worth noting that this path will be absolute

Reply:
Can this be removed? I'm developing some react module and this is the pain to see my name in the dist files.